applications/incident-management/app/src/main/java/lh/quickstart/QuickstartWorkflow.java

```diff
@@ -0,0 +1,56 @@
+package lh.quickstart;
+
+import io.littlehorse.sdk.common.proto.Comparator;
+import io.littlehorse.sdk.common.proto.VariableType;
+import io.littlehorse.sdk.wfsdk.WfRunVariable;
+import io.littlehorse.sdk.wfsdk.Workflow;
+import io.littlehorse.sdk.wfsdk.WorkflowCondition;
+import io.littlehorse.sdk.wfsdk.WorkflowThread;
+import io.littlehorse.sdk.common.proto.VariableMutationType;
+
+public class QuickstartWorkflow {
+
+    // Constants for workflow and task names
+    public static final String WF_Name = "quickstart";
+    public static final String VERIFY_TASK = "verify-incident";
+    public static final String PERIODIC_CHECK_TASK = "periodic-check-task";
+
+    // Method to define the workflow logic
+    public void defineWorkflow(WorkflowThread wf) {
+        // Define workflow variables
+        WfRunVariable incidentSeverity = wf.addVariable("severity", VariableType.INT);
+        WfRunVariable incidentDetails = wf.addVariable("incidentDetails", VariableType.STR);
+        WfRunVariable incidentResolved = wf.addVariable("resolved", VariableType.BOOL);
+
+        // Define condition to check if incident severity is critical (greater than 5)
+        WorkflowCondition isCritical = wf.condition(incidentSeverity, Comparator.GREATER_THAN, 5);
+
+        // Conditional execution: Execute VERIFY_TASK if incident is critical
+        wf.doIf(
+                isCritical,
+                ifBody -> {
+                    ifBody.execute(VERIFY_TASK, incidentDetails);
+                }
+        );
+
+        // Loop execution: Continuously execute PERIODIC_CHECK_TASK until incident is resolved
+        wf.doWhile(
+                wf.condition(incidentResolved, Comparator.EQUALS, false),
+                loopBody -> {
+                    loopBody.execute(PERIODIC_CHECK_TASK, incidentDetails);
```

Comment:
Could we add a `wf.sleep(30)` here to make it sleep 30 seconds? I think that would make some sense, otherwise you will spam the periodic check task (: